packages/flame/lib/src/test_helpers/mock_canvas.dart

```diff
@@ -115,6 +118,34 @@ class MockCanvas extends Fake implements Canvas, Matcher {
     return mismatchDescription.add(matchState['description'] as String);
   }
 
+  bool containsAtLeast(covariant MockCanvas other, Map matchState) {
+    if (_saveCount != 0) {
+      return _fail('Canvas finished with saveCount=$_saveCount', matchState);
+    }
+    final useTolerance = max(tolerance, other.tolerance);
+    final remainingExpectedCommands = other._commands.toList();
+    for (final expectedCommand in _commands) {
```

Comment:
If I'm not mistaken, `this._commands` contains "actual" commands, not expected ones. Well, either that or the other way around, but they can't be both expected :)

Reply:
yes my bad! I got confused during it and forgot to rename one of the instances